src/dump.c

```diff
@@ -2267,6 +2269,13 @@ static void jl_recache_types(void)
     }
 }
 
+extern void (*jl_newmeth_tracer)(jl_method_t *tracee);
+static int trace_method(jl_typemap_entry_t *entry, void *closure)
+{
+    jl_newmeth_tracer(entry->func.method);
```

Comment:
will want to switch this to `jl_call_tracer` when https://github.com/JuliaLang/julia/pull/16040/files#diff-53dc66d2cb199a2ea107533c1f99f57bR78 is merged (adds error handling)